t04_refactored.py

```diff
@@ -128,10 +128,45 @@ def team_2_adv():
 
 ###################################################################################
 
-def team_3_adv():
-    pass
-    # TODO Add your code here
 
+def team_3_adv():
+    """
+    https://docs.google.com/document/d/1Dia-EWKe0B6cUXtTikF31o4zMbq7qywwdQU7uX0No8k/edit\
+    Malak Mohamed
+    Eric Sparks
+    :return: none
+    """
+    print("\n")
+    print("You find a room that contains two gold statues.")
+    statues = input("Do you take the left or right statue, or don't pick up the statue? [left/right/none]: ")
+    print("\n")
+
+    if statues == "left":
+        # good choice
+        print("You pick up the statue on the left.")
+        print("A passageway opens and you see outside.")
+        print("Congrats! You escaped the cave!")
+    elif statues == "right":
+        # bad choice
+        print("You pick up the statue on the right.")
+        print("A passageway opens above you.")
+        print("A bunch of snakes and spiders fall on you!")
+        is_dead = True
+    else:
+        # neutral choice
+        print("You don't pick up either of the statues.")
+        print("Nothing happens... You are still stuck in the cave.")
+    if is_dead:
+        print("\n")
+        print("One of the snakes asks you to pick a number 1 through 20.")
+        number = int(input("Which number do you pick? (only put whole numbers): "))
+        if number >= 13:
+            print("Congrats! The snake decided to spare you!")
+            is_dead = False
+        else:
+            print("You chose a wrong number, which angers the snake. He and the other snakes and spiders bite you!")
+    if is_dead:
+        print("Oh no! You died! Try again by hitting the green play button.")
 ###################################################################################
```

Comment:
At the end of your function, you need to kill your user (possibly) using this function:
`kill_if_dead(dead)`